public/data/qwel-exe.json

```
@@ -5,19 +5,24 @@
   "avatar": "https://github.com/qwe123coder.png",
```

I think you forgot to update this. 😅
```suggestion
  "avatar": "https://github.com/qwel-exe.png",
```